05_generate_report/pipelines/nf-aggregate-pipeline.yml

```
@@ -0,0 +1,12 @@
+pipelines:
+    - name: "nf-aggregate-fusion-benchmark"          
+      url: "https://github.com/seqeralabs/nf-aggregate"
+      workspace: "$ORGANIZATION_NAME/$WORKSPACE_NAME"
+      description: "seqeralabs/nf-aggregate is a Nextflow pipeline to aggregate pertinent metrics across pipeline runs on the Seqera Platform."
+      compute-env: "$COMPUTE_ENV_PREFIX_fusion_nvme"
```

```suggestion
      compute-env: "${COMPUTE_ENV_PREFIX}_fusion_nvme"
```
To avoid the shell looking for the full string as the env var